tests/tests_geomstats/test_visualization_special_euclidean.py

```diff
@@ -0,0 +1,51 @@
+"""Unit tests for visualization."""
+
+import matplotlib
+import matplotlib.pyplot as plt
+
+import geomstats.backend as gs
+import geomstats.visualization as visualization
+import tests.conftest
+from geomstats.geometry.special_euclidean import SpecialEuclidean
+from geomstats.visualization.special_euclidean import SpecialEuclidean3
+
+matplotlib.use("Agg")  # NOQA
+
+
+class TestVisualization(tests.conftest.TestCase):
```

Comment:
TestSpecialEuclidean2

TestSpecialEuclidean3

--> have one test classs per object being tested

Reply:
then, you can remove the suffix "se2" or "se3" because it is clear which viz is being tested